### PR TITLE
Add Bigtable to JSON export

### PR DESCRIPTION
**Pull request by ron-gal**
We want to be able to export from Bigtable to VertexME. The first step is to export from Bigtable to GCS in JSON format. We will create a follow up tool (similar to https://github.com/cloudspannerecosystem/spanner-ai/tree/main) to take the JSON files and send them to VertexME.

### CODE REVIEW

**Comment by bvolpato** [comment]
Thanks Ron! Can you take a look at the failing GH actions before we take a look?

Is an integration test planned?

**Comment by ron-gal** [comment]
Hey Bruno!
It's kind of a WIP, I will for sure fix the broken build and add tests :) Can I move the review to "draft" mode?

**Comment by bvolpato** [comment]
Ack! Sure, there should be a button on github "Still in progress? Convert to draft"

File: v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToJson.java [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+        }
+        jsonWriter.endObject();
+      }
+      jsonWriter.endObject();
+      jsonWriter.endObject();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    return stringWriter.toString();
+  }
```
[the remark this answers is not in the document]
Comment (ron-gal) [reply v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToJson.java:207]: Done

**Pull request merged**